coverage_config_aarch64.json

```diff
@@ -0,0 +1,2 @@
+{"coverage_score": 91.2, "exclude_path": "", "crate_features": ""}
+
```

Comment:
This new line is making the json invalid.

Reply:
weird.. it's (literally) the same file used in the `versionize` crate